config/config_service_presets.go

```diff
@@ -7,18 +7,16 @@
 package config
 
 import (
-	topologyProviderAdapter "github.com/orbs-network/orbs-network-go/services/gossip/adapter"
 	testKeys "github.com/orbs-network/orbs-network-go/test/crypto/keys"
 	"github.com/orbs-network/orbs-spec/types/go/primitives"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/protocol/consensus"
 	"time"
 )
 
-func ForDirectTransportTests(nodeAddress primitives.NodeAddress, gossipPeers topologyProviderAdapter.TransportPeers, keepAliveInterval time.Duration, networkTimeout time.Duration) GossipTransportConfig {
+func ForDirectTransportTests(nodeAddress primitives.NodeAddress, keepAliveInterval time.Duration, networkTimeout time.Duration) GossipTransportConfig {
```

Comment:
I understand why you changed the name of the file but still i would like to see "component" and "test" somewhere in the name: This file contains config presets only for component tests - never used to load a complete node, while the  other file `config_system_presets.go` provides config templates for 1)entire nodes 2) several nodes 2) both prod/dev(gamma) and tests (acceptance).

I suggest naking three files instead - one for component tests, one for system tests, and one for prod/dev:
naming this file `component_tests_presets.go` and extracting acceptance and E2E from the next file to `system_tests_presets.go` while renaming  `config_system_presets.go ` to just `system_presets.go`

Reply:
done